templates/config/grafana.ini.tpl

```diff
@@ -119,10 +119,10 @@ check_for_updates = true
 #################################### Security ####################################
 [security]
 # default admin user, created on startup
-;admin_user = admin
+admin_user = {{.Username}}
```

Comment:
What if the Username is empty?

Reply:
It would be set to `admin` during unmarshal, the default value is set in spec tag